backend/config/environments/development.rb

```diff
@@ -57,6 +76,8 @@
     Bullet.alert = true
     Bullet.bullet_logger = true
     Bullet.console = true
+    # Bullet.growl = true
     Bullet.rails_logger = true
+    Bullet.add_footer = true
```

Comment:
Moved from top of file - `config.after_initialize` Bullet setup was done twice here.